### ISSUE TITLE
Videos aren't downloaded because the post title is not written with LTT tag

### ISSUE THREAD

**Issue by alex73630** [alex73630/Floatplane-Downloader#4]
Those two posts from Computex coverage are not downloading because they don't have any "type" tag on the title.
![image](https://cloud.githubusercontent.com/assets/7538405/26746333/26762a78-47ef-11e7-8cd4-92791d4699ec.png)

This actually doesn't help because that mean that the script need a no type condition, but I don't know where we could add those videos (we can't randomly put them in LTT category).

An option would be to have a webapp where it shows videos that can't be parsed and give some tools to change metadatas in files.

**Comment by aidanappleCO** [comment]
Use their API to get the channel by using the GUID:
(https://cms.linustechtips.com/get/videos/by_guid/GUIDHERE)

Example output is:
`{"id_video":537,"guid":"8D60VAnK3n","title":"Nicky V Airsoft Battle","description":"<p>We do a good old fun airsoft battle! Nicky V attempts to take over the office.</p>","release_type":1,"release_date":null,"added_date":"2017-06-21T00:51:03.000Z","channel":"CSF","creator":"admin","thumbnail":"/get/thumbnail/by_guid/8D60VAnK3n","tags":["Nicky V"]}`

**Comment by alex73630** [comment]
I didn't know about that API, this will help me a lot @aidanappleCO !
I'll rewrite a huge part of my code to implement that instead of parsing the forum page and hope that it'll work.

**Comment by aidanappleCO** [comment]
Yeah I only found that API after trying to reverse engineer Floatplane. Glad it will help.
